test/smoke/testApps/TelemetryProcessors/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/SpringBootAutoTest.java

```diff
@@ -28,10 +28,11 @@ public void doMostBasicTest() throws Exception {
         assertEquals(4, rd.getProperties().size());
         assertTrue(rd.getSuccess());
         // Log processor test
-        List<MessageData> logs = mockedIngestion.getAllMessageData();
+        List<MessageData> logs = mockedIngestion.getMessageDataInRequest();
         for(MessageData log:logs) {
-            if(log.getMessage().equals("This is log from SpringBootApp.")) {
-                assertEquals("testLogAttributeValue", log.getProperties().get("newLogAttribute"));
+            if(log.getProperties().get("LoggerName") != null && log.getProperties().get("LoggerName").equals("smoketestappcontroller")) {
+                System.out.println(log.getMessage());
+                assertEquals("smoketestappcontroller::INFO",log.getMessage());
```

Comment:
can simplify this now, since `getMessageDataInRequest` should only return a single log message